composer.json

```diff
@@ -17,7 +17,7 @@
 
 	"autoload": {
 		"psr-0": {
-			"guymers": "src/main/php"
+			"guymers": ["src/main/php","src/test/php"]
```

Comment:
Instead of always autoloading test files could you instead load them only during tests via the loader in `src/test/resources/bootstrap.php`? https://getcomposer.org/doc/01-basic-usage.md#autoloading